testutil/beaconmock/server.go

```diff
@@ -104,6 +104,12 @@ func newHTTPServer(addr string, overrides ...staticOverride) (*http.Server, erro
 				_, _ = w.Write([]byte(`{"data": {"head_slot": "1","sync_distance": "0","is_syncing": false}}`))
 			},
 		},
+		{
+			Path: "/eth/v1/beacon/headers/head",
+			Handler: func(w http.ResponseWriter, r *http.Request) {
+				_, _ = w.Write([]byte(`{"data": {"header": {"message": {"slot": "1"}}}}`))
+			},
+		},
```

Comment:
why this was needed?

Reply:
teku queries this during voluntary exit to get the current slot for some reason